infrastructure/images/moduletest-reset-jobs/templates/moduletest-file-reset-job.template.yaml

```diff
@@ -0,0 +1,36 @@
+# This manifest will fire up a customer Kubernetes Job in moduletest namepaces.
+# It will synchronize all files from the corresponding main environment to the moduletest environment
+
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: moduletest-reset
+
+---
+
+{{- range .Values.projects }}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: file-reset
```

Comment:
The name will override for each projects, so only one job will run. You should diversify the name based on the project name.

Reply:
Forgot to think about that, thank you for catching that